res/country_metadata/popularReligions.yml

```diff
@@ -7,6 +7,7 @@ AU: [christian, muslim, buddhist]
 AZ: [muslim]
 BD: [muslim, hindu]
 BF: [muslim]
+BG: [christian, muslim]
```

Comment:
Note that it is default already, maybe it should be mentioned somehow in this file?

Either have `default: [christian, muslim, buddhist, hindu, jewish]` or `# note that adding order here is needed only if default order from AddReligionForm needs to be overriden`?

Reply:
It says that the default is `[]` https://github.com/Dimitar5555/StreetComplete/blob/patch-1/res/country_metadata/popularReligions.yml#L3

Reply:
Yes, but it is used solely to change order in https://github.com/streetcomplete/StreetComplete/blob/0857679a146abaf474f051f8e8d12225d9a10ff3/app/src/main/java/de/westnordost/streetcomplete/quests/religion/AddReligionForm.kt#L19 (so default is actually `[christian, muslim, buddhist, hindu, jewish...`)

Reply:
Is it possible to just set the default to `[christian, muslim, buddhist, hindu, jewish...]` in `popularRegligions.yml` or would this break something?

Reply:
it would be possible. I don't think it would break anything.

Reply:
It actually breaks something.

![screen03](https://user-images.githubusercontent.com/899988/144869845-2c39d786-fd55-4ac2-9e6b-92f04f780908.png)

Reply:
Hm but why??

Reply:
I found and fixed problem in https://github.com/streetcomplete/StreetComplete/commit/e8c9b295d598991040ff59dc9044b6f48317a9f0 (in way nicer than my initial attempt but maybe still can be refactored)

The problem was that -1 (not found) was sorted before things at 0th, 1st, 2nd.. positions.